tests/fortran/client_test_dataset.F90

```diff
@@ -195,6 +196,11 @@ program main
   if (result .ne. SRNoError) error stop
   if (.not. all(meta_int64_recv == meta_int64_vec)) error stop 'meta_int64: FAILED'
 
+  ! Test dataset serialization
+  write (*,*) dataset%to_string()
```

Comment:
Might be better to compare this to what the expected string should be just to make sure `to_string` isn't just returning garbage

Reply:
Done